packages/runtime/src/runtimes/react/controls/rich-text.tsx

```diff
@@ -0,0 +1,39 @@
+import { ReactNode, useEffect, useState } from 'react'
+
+import {
+  RichTextControl,
+  RichTextControlData,
+  RichTextControlDefinition,
+} from '../../../controls/rich-text'
+import { PropControllersHandle } from '../../../state/modules/prop-controller-handles'
+import { useIsPreview } from '..'
+import { BoxModelHandle } from '../../../box-model'
+import dynamic from 'next/dynamic'
+import { forwardNextDynamicRef } from '../../../next'
+
+const EditableText = forwardNextDynamicRef(patch =>
```

Comment:
When I instead tried to import the `Text` component here I got some really strange errors.